bfvmm/src/serial/include/bf_portio.h

```diff
@@ -0,0 +1,11 @@
+#ifndef BF_PORTIO__H
+#define BF_PORTIO__H
+
+extern "C" {
```

Comment:
I would wrap these with the cpp macros as a safe guard (just in case they need to be used by C in the future)